packages/e2e/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.2+1
+
+* Remove Android folder from `e2e_macos`.
```

Comment:
The e2e package doesn't need any change. `e2e_macos` is the only one.  @stuartmorgan was this plugin created because e2e wasn't ready to support macos, or is this a federated plugin as well?

Reply:
All the macOS implementations in flutter/plugins are federated.